.github/workflows/deploy.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         version: ["0.5", "0.6"]
     steps:
     - uses: actions/checkout@v2
+    - uses: crazy-max/ghaction-docker-buildx@v1
```

Comment:
The blog post in your PR description seems to be already a bit outdated and this action has been replaced in the meantime by an official one from Docker. [The readme from this action](https://github.com/crazy-max/ghaction-docker-buildx#moved-to-docker-organization) recommends to use this instead:

```yaml
# https://github.com/docker/setup-qemu-action
- name: Set up QEMU
  uses: docker/setup-qemu-action@v1
# https://github.com/docker/setup-buildx-action
- name: Set up Docker Buildx
  uses: docker/setup-buildx-action@v1
```

and these two actions also already have [newer versions](https://github.com/docker/setup-buildx-action#with-qemu):

```yaml
-
  name: Set up QEMU
  uses: docker/setup-qemu-action@v2
-
  name: Set up Docker Buildx
  id: buildx
  uses: docker/setup-buildx-action@v2
```